streams/src/main/java/org/apache/kafka/streams/kstream/internals/KTableKTableInnerJoin.java

```diff
@@ -100,4 +97,38 @@ public void process(final K key, final Change<V1> change) {
         }
     }
 
+    private class KTableKTableInnerJoinValueGetter implements KTableValueGetter<K, R> {
+
+        private final KTableValueGetter<K, V1> valueGetter1;
+        private final KTableValueGetter<K, V2> valueGetter2;
+
+        KTableKTableInnerJoinValueGetter(final KTableValueGetter<K, V1> valueGetter1,
+                                         final KTableValueGetter<K, V2> valueGetter2) {
+            this.valueGetter1 = valueGetter1;
+            this.valueGetter2 = valueGetter2;
+        }
+
+        @Override
+        public void init(ProcessorContext context) {
+            valueGetter1.init(context);
+            valueGetter2.init(context);
+        }
+
+        @Override
+        public R get(K key) {
+            R newValue = null;
```

Comment:
Can we remove `newValue` and do `return null` and `return joiner.apply(value1, value2);` instead?

Reply:
ack